website/src/components/Sidebar.js

```diff
@@ -1,10 +1,28 @@
-import React from 'react'
+import React, { useState } from 'react'
 
 function Sidebar ({children = []}) {
+    const [filter, setFilter] = useState("");
+
     return (
         <div className="md:relative fixed animate__animated -left-full md:left-0 dark:bg-gray-700 bg-white shadow-md h-full md:block md:pt-0 md:w-4/12 top-0 w-9/12 z-20" id="sidebar">
             <div className="md:fixed h-full md:h-4/6 md:pt-0 md:w-4/12 overflow-y-auto pb-3 pl-5 pt-5">
-                {children}
+                <div className="flex pb-5">
+                    <label for="search" className="">Search:</label>
+                    <input 
+                        name="search" type="text"
+                        value={filter}
+                        onChange={(e) => setFilter(e.target.value)}
+                        className="ml-2 border-2 border-gray-400 focus:border-red-600"></input>
```

Comment:
Change classes to `border focus:border-red-600 focus:outline-none pl-3`.

Reply:
This has been updated as requested.